app/styles/_layout.scss

```diff
@@ -6,7 +6,9 @@
   text-align: center;
 
   &.card {
-    width: 420px;
+    @include respond-to('big') {
```

Comment:
@shane-tomlinson oh, yeah, that's probably the most sensible thing! A+ for efficiency!